src/com/t_oster/visicut/model/graphicelements/svgsupport/SVGImporter.java

```diff
@@ -237,22 +267,26 @@ public double determineResolution(File f, List<String> warnings)
     // SVG files loaded in inkscape retain the Illustrator comment, but are saved with 90 DPI.
     if (AdobeIllustratorSeen) { result = 72; }
     if (WwwInkscapeComSeen) { result = 90; }	// inkscape wins over Illustrator
+    if (InkscapeVersion092Seen) { result = 96; }	// inkscape with known version wins over anything else.
     if (result != 90)
     {
        if (AdobeIllustratorSeen)
          {
            warnings.add("Adobe Illustrator comment seen in SVG.");
 	 }
-       warnings.add("Switching to DPI from 90 to " + result + " - Please check object size!");
-
+       if (InkscapeVersion092Seen)
+         {
+           warnings.add("Inkscape Version 0.92+ comment seen in SVG.");
+	 }
+       warnings.add("Switching DPI from 90 to " + result + " - Please check object size!");
     }
     return result;
   }
 
   /*
    * Tries to determine the Coordinate resolution in DPI.
    * SVG default is 90, but AI generates 72??
```

Comment:
Oh jeah, I meant to include all 1.x version of inkscape too. Thakns for catching this.

The SVG standard consortium wronly decided to change the existing standard instead of releasing a new version of the standard.

The warning messages are only intended to be there temprorarily to raise awareness of the issue. I am pretty sure there are cornercases, that we cannot catch.